article-api/src/main/scala/no/ndla/articleapi/service/ReadService.scala

```diff
@@ -79,6 +79,13 @@ trait ReadService {
       }
     }
 
+    def getFrontpageArticle(slug: String, language: String, fallback: Boolean = false): Try[api.ArticleV2] = {
```

Comment:
En kan vel tenke seg at vi kan utvide med nye typer artikler som kan ha slug. Kanskje til og med at vi kan trenge slug for andre artikler også...
```suggestion
    def getBySlug(slug: String, language: String, fallback: Boolean = false): Try[api.ArticleV2] = {
```